### PR TITLE
fix csv export naming

### PR DESCRIPTION
**Pull request by sanyud24**
closes https://github.com/odk-x/tool-suite-X/issues/333

### CODE REVIEW

**Comment by wbrunette** [comment]
runtests

File: src/main/java/org/opendatakit/suitcase/utils/FileUtils.java [hunk 1]
```diff
@@ -123,15 +123,17 @@ public static Path getInstancesPath(CloudEndpointInfo cloudEndpointInfo, String
...
+  public static String getCSVName(CsvConfig config, String tableId) {
     StringBuilder csvNameBuilder = new StringBuilder();
 
     csvNameBuilder
-        .append(config.isDownloadAttachment() ? DATA_CSV_MODIFIER : LINK_CSV_MODIFIER);
-
-    csvNameBuilder
-        .append(CSV_MODIFIER_SEPARATOR)
-        .append(config.isScanFormatting() ? FORMATTED_CSV_MODIFIER : UNFORMATTED_CSV_MODIFIER);
+        .append(config.isDownloadAttachment() ? DATA_CSV_MODIFIER : tableId);
```
Comment (wbrunette) [inline src/main/java/org/opendatakit/suitcase/utils/FileUtils.java:130]: Now looking at how the code works, it is best to leave the LINK_CSV_MODIFIER name and instead simply append the table tableId before line 128 and append a CSV_MODIFIER_SEPARATOR.

So after line 127 it should be something like "csvNameBuilder.append(tableId).append(CSV_MODIFIER_SEPARATOR).

**Review by wbrunette — CHANGES_REQUESTED** [review]

**Comment by wbrunette** [comment]
@sanyud24, great job! When creating the issue, I had not looked at the code. After reviewing your change, I now understand better, so I have a suggested revision.

File: src/main/java/org/opendatakit/suitcase/utils/FileUtils.java (hunk 1 above)
Comment (sanyud24) [reply src/main/java/org/opendatakit/suitcase/utils/FileUtils.java:130]: Sure !! will make the changes

**Comment by sanyud24** [comment]
> @sanyud24, great job! When creating the issue, I had not looked at the code. After reviewing your change, I now understand better, so I have a suggested revision.

Thankyou :relaxed:

File: src/main/java/org/opendatakit/suitcase/utils/FileUtils.java [hunk 2]
```diff
@@ -123,18 +123,18 @@ public static Path getInstancesPath(CloudEndpointInfo cloudEndpointInfo, String
     return getBasePath(cloudEndpointInfo, tableId, savePath).resolve(INSTANCES_PATH);
   }
 
-  public static String getCSVName(CsvConfig config) {
+  public static String getCSVName(CsvConfig config, String tableId) {
     StringBuilder csvNameBuilder = new StringBuilder();
```
Comment (wbrunette) [inline src/main/java/org/opendatakit/suitcase/utils/FileUtils.java:127]: The tableId should be at the beginning so it is added after line 127.

**Review by wbrunette — CHANGES_REQUESTED** [review]

File: src/main/java/org/opendatakit/suitcase/utils/FileUtils.java (hunk 2 above)
Comment (sanyud24) [reply src/main/java/org/opendatakit/suitcase/utils/FileUtils.java:127]: Ohkay

**Review by wbrunette — APPROVED** [review]

**Pull request merged**